tests/components/azure_data_explorer/test_config_flow.py

```diff
@@ -0,0 +1,124 @@
+"""Test the AEH config flow."""
+import logging
+
+from azure.kusto.data.exceptions import KustoAuthenticationError, KustoServiceError
+import pytest
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.components.azure_data_explorer.const import DOMAIN, STEP_USER
+
+from .const import BASE_CONFIG, UPDATE_OPTIONS
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    ("step_config", "step_id"),
+    [
+        (BASE_CONFIG, STEP_USER),
+    ],
+    ids=["Base"],
+)
+async def test_form(
+    hass,
+    mock_setup_entry,
+    step_config,
+    step_id,
+) -> None:
+    """Test we get the form."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}, data=None
+    )
+    assert result["type"] == "form"
```

Comment:
```suggestion
    assert result["type"] == FlowResultType.FORM
```

Please adopt all other places in your tests too